hare/builder.go

```diff
@@ -1,19 +1,49 @@
 package hare
 
 import (
-	"github.com/gogo/protobuf/proto"
-	"github.com/spacemeshos/go-spacemesh/hare/pb"
+	"bytes"
+	"github.com/nullstyle/go-xdr/xdr3"
 	"github.com/spacemeshos/go-spacemesh/log"
 )
 
-// Used to build proto messages
+// top Message of the protocol
+type XDRMessage struct {
```

Comment:
Why not  name it just message? the encoding method is irrelevant

Reply:
Fixed